README.md

```diff
@@ -1,45 +1,101 @@
 # Normalized Get
 
-A `get` function that traverses normalizr relationships.
+Helper functions for traversing [normalizr](https://github.com/paularmstrong/normalizr) relationships.
 
 ## Who is this for?
 
 Anyone that uses [normalizr](https://github.com/paularmstrong/normalizr), or normalized data that conforms to it.
 
 ## What does it do?
 
-Let's you access normalized data as though it was denormalized. 
+1. `NormalizedGet` lets you access normalized data using a path as though it was denormalized. 
+2. `GraphGet` lets you denormalize mutliple paths at once, perfect for using in components.
+
+### NormalizedGet
 
 Say you have a schema like this:
 ```js
   const users = new schema.Entity('users');
+  const comments = new schema.Entity('comments', {
+    user: users
+  });
   const articles = new schema.Entity('articles', {
-    author: users
+    author: users,
+    comments: [comments]
   });
-  const schemas { articles, users };
+  return { comments, articles, users };
+```
+
+Just setup `bindNoramlizedGet` like so:
+```js
+const normlizedGet = bindNormalizedGet(schemas, data); 
```

Comment:
should be `normalizedGet`. looks like this typo is all over the place, maybe find/replace would fix it everywhere